brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-667 New - Box Shadow and Inner Container preset implementation in blocks.

### Description

- Implemented Box Shadow preset in following blocks ( Container, Buttons child, Taxonomy List, Image ).
- Implemented Inner Container preset for Container block.

### Screenshots


### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop any block mentioned above.
- Go to Box Shadow setting, Set, or click on any box-shadow preset.
- Check it is applying in the editor as well as on the front same as preset.
- For Inner Container Preset drag and drop container block.
- Check the Preset tab, Change preset and check it is working fine on the frontend as well as on the editor.

### Checklist:
